bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/MenuItem.java

```diff
@@ -47,7 +47,8 @@ public class MenuItem extends Item {
 	/* Image margin. */
 	final static int MARGIN_WIDTH = 1;
 	final static int MARGIN_HEIGHT = 1;
-
+	final static int LEFT_TEXT_MARGIN = 15;
+	final static int ADDED_ITEM_WIDTH = 15;
```

Comment:
How are these values derived? The margins look rather different to the existing implementation and to the light theme. In the existing implementation, a value of "5" was hardcoded for the left margin at some point, maybe that would fit as well?

Reply:
I am going back to the value 5 for item width as before (because of larger gap) and left margin value was derived as a result of testing against different values and finding out the closest to the light theme.